cmd/zt_copy_blob_upload_test.go

```diff
@@ -29,6 +29,84 @@ import (
 	"strings"
 )
 
+func (s *cmdIntegrationSuite) TestIncludeDir(c *chk.C) {
+	bsu := getBSU()
+	containerURL, containerName := createNewContainer(c, bsu)
+	defer deleteContainer(c, containerURL)
+
+	files := []string{
+		"filea",
+		"fileb",
+		"filec",
+		"sub/filea",
+		"sub/fileb",
+		"sub/filec",
+		"sub/subsub/filea",
+		"sub/subsub/fileb",
+		"sub/subsub/filec",
+	}
```

Comment:
should probably also have a case here `sub/somethingelse/subsub/filex`  (which I presume would not be included)